warehouse/models/mart/gtfs_quality/fct_monthly_reports_site_organization_gtfs_vendors.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(materialized='table') }}
+
+WITH assessed_entities AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_quality__daily_assessment_candidate_entities') }}
+    -- filter to only organizations that are supposed to be assessed
+    WHERE reports_site_assessed
+),
+
+gtfs_generation_components AS (
+    SELECT *
+    FROM {{ ref('dim_service_components') }}
+    WHERE component_name IN
+        ('GTFS generation', 'Scheduling (Fixed-route)', 'Real-time info',
+        'Arrival predictions', 'GTFS-rt vehicles/trips', 'GTFS Alerts Publication')
+),
+
+dim_orgs AS (
+    SELECT
+        source_record_id, name, organization_type
+    FROM {{ ref('dim_organizations') }}
+    WHERE _is_current
```

Comment:
Just noting that by adding this filter, you *might* risk having a failed join if an organization record for a vendor has been deleted by was active in the past. Don't think it's a huge risk because IIRC the service components tables aren't fully historical yet (which we may want to just double check on..... If there is anyone we know has changed vendors, I think we may need to look into making those fully historical so we aren't assessing a current vendor on a prior vendor's data?)

Reply:
Put another way, I think we might eventually need to make a vendor/components bridge table that uses the versioned `key`s that you could use here 🤔

Reply:
So, I think this is probably ok for now, but maybe we need a ticket to make it more robust in future?